config/locales/en.yml

```diff
@@ -36,11 +36,15 @@ en:
         name: Group name
         path: Group slug
         parent_id: Group URL
+        one: Group
```

Comment:
These need to be under `models` and not `attributes`, please re-refer to comment above.

Reply:
Updated in 20e119965680ad3aa0f927bffc00468e33c0fff7